appveyor.yml

```diff
@@ -9,6 +11,12 @@ environment:
     - PYTHON: "C:\\Python36"
       TOX_ENV: "py36"
 
+  global:
+    LOGNAME: "LOGNAME"
```

Comment:
Actually, this was on purpose because of GitPython. A better approach was taken on commit https://github.com/ESSS/module-renamer/commit/0070645ed6494cb511f8b59d9032310f8eac718c . The file ` tox.ini` has comments and links explaining the issue that these variables solve.

- [ x ] Task: Remove global env